examples/server-customobject.py

```diff
@@ -16,28 +22,44 @@
     uri = "http://examples.freeopcua.github.io"
     idx = server.register_namespace(uri)
 
-    # Import customobject type
-    server.import_xml('customobject.xml')
-
     # get Objects node, this is where we should put our custom stuff
     objects = server.get_objects_node()
 
+    # Example 1 - create a basic object
+    #-------------------------------------------------------------------------------
+    myobj = objects.create_object(idx, "MyObject",)    
+    #-------------------------------------------------------------------------------
+
+
+    # Example 2 - create a new object type and a instance of the new object type
+    #-------------------------------------------------------------------------------
+    types = server.get_node(ua.ObjectIds.BaseObjectType)
```

Comment:
i would write this like
```
server.create_object_type(create_custom_object_type, idx, "MyCustomObject")
```

In this way you "sqash" two lines into one. And this is meant to be used by the users since protects from doing unintended errors.